diff --git a/dist/chunk-UL7Y3R6S.js b/dist/chunk-UL7Y3R6S.js
index a9bb1effc18c1ca99d01fcd28846e9ddec90a308..4ed948f6a0990c4e71d1e57b415c3f7cf183676a 100644
--- a/dist/chunk-UL7Y3R6S.js
+++ b/dist/chunk-UL7Y3R6S.js
@@ -983,7 +983,7 @@ export default {
 
 
 async function getPackageData(pkg) {
-  const { resolvePackageData } = await Promise.resolve().then(() => require("vite"));
+  const { resolvePackageData } = await import("vite");
   const info = resolvePackageData(pkg, process.cwd());
   if (!info)
     return;